### PR TITLE
Add html file and css file with linters

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
>All good
-index.html added in h1 heading with the attribute class 
-style.css added in one class for h1 element in index.html
-fixed linter error

### CODE REVIEW

**Comment by kinginthenorthcodez** [comment]
Lint error fixed in style due to file path

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+3. When you open your first pull request you should see the result of the GitHub actions checks:
+
+![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+
+Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+
+## Set-up linters in your local env
+
```
Comment (Whoistolu) [inline README.md]: - [ ] Kindly ensure that your README.md file is descriptive. Please remove all the unnecessary info within the README that does not pertain to this project.
Kindly use this [Template](https://github.com/microverseinc/readme-template)

File: node_modules/.bin/acorn [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+case `uname` in
+    *CYGWIN*|*MINGW*|*MSYS*) basedir=`cygpath -w "$basedir"`;;
+esac
+
+if [ -x "$basedir/node" ]; then
+  exec "$basedir/node"  "$basedir/../acorn/bin/acorn" "$@"
+else 
+  exec node  "$basedir/../acorn/bin/acorn" "$@"
+fi
```
Comment (Whoistolu) [inline node_modules/.bin/acorn:12]: - [ ] Kindly ensure the `node_modules` and the `test.md` files are ignored by the `gitignore`.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @kinginthenorthcodez ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the inline comments under the review for sections to improve._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise_.

File: README.md (hunk 1 above)
Comment (kinginthenorthcodez) [reply README.md]: Updated REAME.md  to use the template

File: node_modules/.bin/acorn (hunk 2 above)
Comment (kinginthenorthcodez) [reply node_modules/.bin/acorn:12]: Added node_modules and test.md in gitignore

**Review by Deepakdanger — APPROVED** [review]
Hi @kinginthenorthcodez , 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- We never push the node_modules folder to the GitHub repo due to its large size. So always include node_modules in `.gitignore`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
> Hi @kinginthenorthcodez ,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it 🚢 Congratulations! 🎉
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
> * We never push the node_modules folder to the GitHub repo due to its large size. So always include node_modules in `.gitignore`
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

>Hi @Deepakdanger  thank you for that quick and heads-up review on the node_modules dir
>
>Cheers and Happy coding too!

**Comment by kinginthenorthcodez** [comment]
>merged hml-css-linters to main branch

**Pull request merged**